src/popup/components/Filters.vue

```diff
@@ -35,15 +36,26 @@ export default {
   props: {
     value: { type: Object, required: true },
     filters: { type: Object, required: true },
-    fixed: Boolean,
+    stickyTopHeight: { type: Number, default: 0 },
+    sticky: Boolean,
```

Comment:
There's a rule that I red once and liked a lot, that says the component should not position itself, the parent scope should control the positioning of child elements.

So I have this crazy idea: maybe the filters should not have the `sticky` prop? Then the parent would organise the elements and position the filters (make it sticky). The benefit would be encapsulation of the CSS code that controls the sizes and the dependancies in offsets.

What do you think?